features/gba-dev/waardetabel-gba.feature

```diff
@@ -0,0 +1,247 @@
+
+      Abstract Scenario: code voor <element> komt niet voor in de tabel <tabel>
+        Gegeven de persoon met burgerservicenummer '000000231' heeft de volgende gegevens
+        | <element> |
+        | <waarde>  |
+        Als gba personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000231                       |
+        | fields              | <groep>.<veld>                  |
+        Dan heeft de response een persoon met alleen de volgende '<groep>' gegevens
+        | naam                | waarde         |
+        | <veld>.code         | <waarde>       |
+
+        Voorbeelden:
+        | element                           | tabel               | groep                        | veld                    | waarde |
+        | geboorteplaats (03.20)            | Gemeenten           | geboorte                     | plaats                  | 9876   |
+        | geboorteland (03.30)              | Landen              | geboorte                     | land                    | 1234   |
+
+      Abstract Scenario: code voor <element> komt niet voor in de tabel <tabel>
+        Gegeven de persoon met burgerservicenummer '000000243' heeft de volgende '<groep>' gegevens
+        | <element> |
+        | <waarde>  |
+        Als gba personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000243                       |
+        | fields              | <groep>                         |
+        Dan heeft de response een persoon met alleen de volgende '<groep>' gegevens
+        | naam                | waarde         |
+        | <veld>.code         | <waarde>       |
+
+        Voorbeelden:
+        | element                           | tabel               | groep          | veld                    | waarde |
+        | plaats overlijden (08.20)         | Gemeenten           | overlijden     | plaats                  | 9876   |
+        | land overlijden (08.30)           | Landen              | overlijden     | land                    | 1234   |
+        | aanduiding verblijfstitel (39.10) | Verblijfstiteltabel | verblijfstitel | aanduiding              | 01     |
+        | land adres buitenland (13.10)     | Landen              | verblijfplaats | land                    | 1234   |
+        | land vanwaar ingeschreven (14.10) | Landen              | immigratie     | landVanwaarIngeschreven | 1234   |
+        # In het laatste voorbeeld zou eigenlijk de "groep" immigratie moeten zijn. Echter is dit nog niet geïmplementeerd in de automation code. Als dit correct geïmplementeerd is kan de kolom "fields" vervallen en kan daarvoer "groep" gebruikt worden
```

Comment:
Kan deze opmerking al komen te vervallen? De kolom 'fields' komt immers niet voor.

Reply:
De voorbeelden op regel 178 en 179 zijn komen te vervallen. Die zijn immer in de scenario dat daarna komt opgenomen. Ik was vergeten ze te verwijderen. De opmerking die jij aangeeft hoort dan ook bij dat volgende scenario.